### PR TITLE
Update README for Remark JS

### PR DESCRIPTION
**Pull request by sunny**
Fixes README following comment on issue https://github.com/astefanutti/decktape/issues/157#issuecomment-497517601

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -331,11 +331,11 @@ Arch Linux, among other Linux distributions may have the user namespace in the k
 * *_Failed to read the 'rules' property from 'CSSStyleSheet': Cannot access rules_*
 +
 Starting Chromium 64, accessing CSS rules in a stylesheet loaded from the local filesystem violates a CORS policy.
```
Comment (astefanutti) [inline README.adoc]: I'd suggest to change:

"_loaded from the local filesystem violates a CORS policy_"

With:

"_loaded from the local filesystem or an external location violates CORS policies_"

File: README.adoc [hunk 2]
```diff
@@ -331,11 +331,11 @@ Arch Linux, among other Linux distributions may have the user namespace in the k
...
-As some Decktape plugins tweak the CSS rules for better PDF printing, you need to allow access to local files by setting the `--allow-file-access-from-files` flag option, e.g.:
+As some Decktape plugins tweak the CSS rules for better PDF printing, you need to allow access to local files by setting the `--disable-web-security` flag option, e.g.:
```
Comment (astefanutti) [inline README.adoc]: "_allow access to local files_" -> "_allow access to local files or external stylesheets_"

File: README.adoc (hunk 1 above)
Comment (sunny) [inline README.adoc]:
```suggestion
Starting Chromium 64, accessing CSS rules in a stylesheet loaded from the local filesystem or an external location violates CORS policies.
```

File: README.adoc (hunk 2 above)
Comment (sunny) [inline README.adoc]:
```suggestion
As some Decktape plugins tweak the CSS rules for better PDF printing, you need to allow access to local files or external stylesheets by setting the `--disable-web-security` flag option, e.g.:
```

**Comment by sunny** [comment]
Much better, indeed!

**Comment by astefanutti** [comment]
Thanks a lot!

**Pull request merged**